jsr352/core/src/main/java/org/hibernate/search/jsr352/massindexing/MassIndexingJobAvailableParameters.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.jsr352.massindexing;
+
+
+/**
+ * @author Yoann Rodiere
+ */
+public final class MassIndexingJobAvailableParameters {
```

Comment:
>>MassIndexing is used in the package name, so when users use this class, they know the class for mass-indexing.
>
>For utility classes I prefer not to rely on the package name to differentiate utilities: it messes with IDE auto-completion when you have multiple such classes with the same name, making it harder to use them. See for instance AvailableSettings, you have 2 or 3 of them and it's frankly annoying. On top of that, in your code you don't see the package name right away, so it may end up quite confusing.

+1.